bitnami/redis/values.yaml

```diff
@@ -921,6 +921,9 @@ podSecurityPolicy:
   ## Specifies whether a PodSecurityPolicy should be created
   ##
   create: false
+  ## Enable PodSecurityPolicy
+  ##
+  enabled: false
```

Comment:
Same as before: please, move this above `create:` and update the `create` description to mention that you should also set `podSecurityPolicy.enabled` to true to enable it.